eth/fetcher/block_fetcher.go

```diff
@@ -903,7 +905,7 @@ func (f *BlockFetcher) importBlocks(op *blockOrHeaderInject) {
 		}
 
 		if block.Header().EmptyWithdrawalsHash() {
-			block = block.WithWithdrawals(make([]*types.Withdrawal, 0))
+			block = block.WithWithdrawals(make([]*types.Withdrawal, 0)).WithReceiveInfos(block.ReceivedAt, block.ReceivedFrom)
```

Comment:
.WithReceiveInfos(block.ReceivedAt, block.ReceivedFrom) ?
assign fields of self to self?